tests/integration/test_fim/test_registry/test_registry_file_limit/test_registry_limit_values.py

```diff
@@ -54,37 +54,35 @@
 tags:
     - fim_registry_file_limit
 '''
-import os
 
+import os, sys
 import pytest
 from wazuh_testing import global_parameters
-from wazuh_testing.fim import LOG_FILE_PATH, callback_value_file_limit, generate_params, \
-    callback_registry_count_entries, modify_registry_value, registry_parser, KEY_WOW64_64KEY, REG_SZ, KEY_ALL_ACCESS, \
-    RegOpenKeyEx, RegCloseKey
-from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
-from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.fim import (LOG_FILE_PATH, generate_params, modify_registry_value, registry_parser, KEY_WOW64_64KEY,
+                               REG_SZ, KEY_ALL_ACCESS, RegOpenKeyEx, RegCloseKey, create_registry)
+from wazuh_testing.fim_module.fim_variables import (WINDOWS_HKEY_LOCAL_MACHINE, MONITORED_KEY, CB_FILE_LIMIT_VALUE,
+                                                    ERR_MSG_FILE_LIMIT_VALUES, CB_COUNT_REGISTRY_FIM_ENTRIES,
+                                                    ERR_MSG_FIM_INODE_ENTRIES, ERR_MSG_WRONG_NUMBER_OF_ENTRIES,
+                                                    ERR_MSG_WRONG_FILE_LIMIT_VALUE)
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import FileMonitor, callback_generator
 
 # Marks
 
-
 pytestmark = [pytest.mark.win32, pytest.mark.tier(level=1)]
 
 # Variables
 
-
-KEY = "HKEY_LOCAL_MACHINE"
-sub_key_1 = "SOFTWARE\\test_key"
-
+KEY = WINDOWS_HKEY_LOCAL_MACHINE
+sub_key_1 = MONITORED_KEY
 test_regs = [os.path.join(KEY, sub_key_1)]
+reg1 = test_regs[0]
 test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
 wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
-reg1 = test_regs[0]
 
 # Configurations
 
-
 file_limit_list = ['1', '10', '100', '1000']
-
 conf_params = {'WINDOWS_REGISTRY': reg1, 'MODULE_NAME': __name__}
 p, m = generate_params(extra_params=conf_params,
```

Comment:
changes the variables in order to representative
```suggestion
conf_params, conf_metadata = generate_params(extra_params=conf_params,
```

Reply:
Fixed in commit https://github.com/wazuh/wazuh-qa/pull/2478/commits/50987ed612082b485cdda2af0c6f3c1aeceffed0